### DISCUSSION TITLE
snapshot mirror pod has error

### DISCUSSION THREAD

**Discussion by 13567436138** [rook/rook#13509] [Q&A]
cluster2:
debug 2024-01-05T06:45:50.891+0000 7f8e0c7fd700 -1 cephfs::mirror::PeerReplayer(afc319ba-b4b3-4c94-b3de-1ca344b1ce6a) build_snap_map: snap_path=//.snap/scheduled-2024-01-04-09_00_00_UTC has invalid metadata in remote snapshot
debug 2024-01-05T06:45:50.891+0000 7f8e0c7fd700 -1 cephfs::mirror::PeerReplayer(afc319ba-b4b3-4c94-b3de-1ca344b1ce6a) do_sync_snaps: failed to build remote snap map
debug 2024-01-05T06:45:50.891+0000 7f8e0c7fd700 -1 cephfs::mirror::PeerReplayer(afc319ba-b4b3-4c94-b3de-1ca344b1ce6a) sync_snaps: failed to sync snapshots for dir_root=/

cluster2:
imed out
debug 2024-01-05T03:24:10.798+0000 7f1c95fb4700 -1 cephfs::mirror::Mirror handle_enable_mirroring: failed to initialize FSMirror for filesystem={fscid=1, fs_name=myfs}: (110) Connection timed out
debug 2024-01-05T03:24:22.978+0000 7f1c95fb4700 -1 cephfs::mirror::Utils mount: mount error: (110) Connection timed out
debug 2024-01-05T03:24:22.982+0000 7f1c95fb4700 -1 cephfs::mirror::Mirror handle_enable_mirroring: failed to initialize FSMirror for filesystem={fscid=1, fs_name=myfs}: (110) Connection timed out
debug 2024-01-05T03:58:02.359+0000 7f1c4c721700  0 client.54335 ms_handle_reset on v2:192.168.229.186:3300/0
debug 2024-01-05T03:58:07.359+0000 7f1c4c721700  0 client.54335 ms_handle_reset on v2:192.168.229.186:3300/0
debug 2024-01-05T05:25:18.205+0000 7f1c4c721700  0 client.54335 ms_handle_reset on v2:192.168.229.186:3300/0
debug 2024-01-05T05:25:23.217+0000 7f1c4c721700  0 client.54335 ms_handle_reset on v2:192.168.229.186:3300/0

**Comment by zhucan** [comment]
the connection timed out,  maybe you need to check the network is reachable？

**Reply by 13567436138** [reply]
connect from what to what，when mirror
